cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -243,6 +247,21 @@ func (s *CMServer) Run(_ []string) error {
 	}
 	pvRecycler.Run()
 
+	var rootCAProvider func() []byte
+
+	if s.RootCAFile != "" {
+		rootCA, err := ioutil.ReadFile(s.RootCAFile)
+		if err != nil {
+			return fmt.Errorf("error reading root-ca-file: %v", err)
```

Comment:
nit: I always find it helpful to print the name of the file in addition to the error.